src/opera/test/pge/dswx_ni/test_dswx_ni_pge.py

```diff
@@ -533,7 +534,7 @@ def test_iso_metadata_creation(self):
             yaml.safe_dump(runconfig_dict, outfile, sort_keys=False)
 
         try:
-            pge = DSWxNIExecutor(pge_name="DswxS1PgeTest", runconfig_path=test_runconfig_path)
+            pge = DSWxNIExecutor(pge_name="DswxNiPgeTest", runconfig_path=test_runconfig_path)
```

Comment:
👍

Reply:
The was probably my error.  Thanks for catching i